azurerm/resource_arm_batch_pool_test.go

```diff
@@ -323,7 +324,8 @@ resource "azurerm_batch_pool" "test" {
   resource_group_name = "${azurerm_resource_group.test.name}"
   account_name        = "${azurerm_batch_account.test.name}"
   display_name        = "Test Acc Pool"
-  vm_size             = "Standard_A1"
+	vm_size             = "Standard_A1",
```

Comment:
Could we fix the formatting/alignment here?

Reply:
Done